supporting/shims/kiennq/version.txt

```diff
@@ -1 +0,0 @@
-2.2.1
```

Comment:
Would be better to keep the file indicating what version is vendored.

Reply:
Readd it